client/components/Players/PlayerManager.vue

```diff
@@ -84,7 +84,7 @@
                       <v-btn
                         v-if="supportedFeatures.pictureInPicture"
                         icon
-                        disabled
+                        @click="togglePictureInPicture"
                       >
                         <v-icon> mdi-picture-in-picture-bottom-right </v-icon>
```

Comment:
```suggestion
                        <v-icon>mdi-picture-in-picture-bottom-right</v-icon>
```

Reply:
I'm stupid, it's clearer this way ^^

Reply:
Ah no worried. There were a bunch of `v-icons` that had trailing spaces, so I have removed them all in a separate commit.